src/__tests__/surveys.js

```diff
@@ -101,6 +101,13 @@ describe('surveys', () => {
         expect(given.instance._send_request).toHaveBeenCalledTimes(2)
     })
 
+    it('getSurveys returns empty array if surveys are undefined', () => {
+        given('surveysResponse', () => ({ surveys: undefined }))
```

Comment:
This occurs when the request errors out, which is why there's nothing in surveys. As in other cases, our API always returns an array.

```suggestion
        given('surveysResponse', () => ({ status: 0 }))
```